autogpt_platform/frontend/src/tests/pages/navbar.page.ts

```diff
@@ -0,0 +1,51 @@
+import { Page } from "@playwright/test";
+
+export class NavBar {
+  constructor(private page: Page) {}
+
+  async clickProfileLink() {
+    // await this.page.getByTestId("profile-link").click();
+
+    await this.page.getByRole("button", { name: "CN" }).click();
+    await this.page.getByRole("menuitem", { name: "Profile" }).click();
+  }
+
+  async clickMonitorLink() {
+    await this.page.getByTestId("monitor-link").click();
+  }
+
+  async clickBuildLink() {
+    await this.page.getByTestId("build-link").click();
+  }
+
+  async clickMarketplaceLink() {
+    await this.page.getByTestId("marketplace-link").click();
+  }
```

Comment:
These links aren't actually implemented yet. Will setup on shadcn to do these if I have time. If not, will manually update as needed.